src/runtime/composables/turnstile.ts

```diff
@@ -0,0 +1,19 @@
+import { useScript } from "#imports"
+
+declare global {
+    interface Window {
+        turnstile: Turnstile.Turnstile
+    }
+}
+
+export function useScriptCloudflareTurnstile() {
```

Comment:
If it's intended that the user only interact with the module via the component we may be safe to just handle this logic in the component.

The problem we have now is we should be providing a way for users to modify the options, specifically they should be able to change the `trigger` and `assetStrategy`.

Reply:
In that case, should we remove the mocked `useScriptCloudflareTurnstile` in `@nuxt/scripts` ?

Reply:
Yes this is now done